src/app/archive/archive_lib/types.ml

```diff
@@ -296,12 +296,12 @@ module Blocks = struct
         (External_transition.t, State_hash.t) With_hash.t)
       (user_commands :
         ( (Coda_base.User_command.t, Transaction_hash.t) With_hash.t
```

Comment:
If `Coda_base` isn't needed for `Block_time`, can it also be omitted for `User_command` here?